_includes/head.html

```diff
@@ -1,17 +1,8 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  {% if page.title == "Home" %}
-    <meta name="description" content="{{ site.description }}">
-  {% endif %}
 
-  <title>
-    {% if page.title == "Home" %}
-      {{ site.title }}
-    {% else %}
-      {{ page.title }} &middot; {{ site.title }}
-    {% endif %}
-  </title>
+  {% seo %}
```

Comment:
May I know what this line does?

Reply:
The [SEO tag](https://github.com/jekyll/jekyll-seo-tag/blob/master/docs/usage.md) for Jekyll will automatically generate some HTML elements such as `title`, `description`, `author` etc.

Below is an example of what `{% seo %}` would output for https://chesterhow.github.io/tale

```html
<!-- Begin Jekyll SEO tag v2.5.0 -->
<title>Home | Tale</title>
<meta name="generator" content="Jekyll v3.7.4" />
<meta property="og:title" content="Home" />
<meta name="author" content="Chester How" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Minimal Jekyll theme for storytellers" />
<meta property="og:description" content="Minimal Jekyll theme for storytellers" />
<link rel="canonical" href="https://chesterhow.github.io/tale/" />
<meta property="og:url" content="https://chesterhow.github.io/tale/" />
<meta property="og:site_name" content="Tale" />
<link rel="next" href="https://chesterhow.github.io/tale/page2" />
<script type="application/ld+json">
  {
    "@type":"WebSite",
    "url":"https://chesterhow.github.io/tale/",
    "name":"Tale",
    "headline":"Home",
    "author":{"@type":"Person","name":"Chester How"},
    "description":"Minimal Jekyll theme for storytellers",
    "@context":"http://schema.org"
  }
</script>
<!-- End Jekyll SEO tag -->
```

Reply:
Also, if you do not like the way `{% seo %}` outputs the title, the following line can be used to disable title generation: `{% seo title=false %}`.

Reply:
I see.. that's pretty handy!